package.json

```diff
@@ -58,12 +58,12 @@
     "@angular/router": "~7.0.3",
     "@angular/upgrade": "~7.0.3",
     "@hmcts/ccd-case-ui-toolkit": "2.62.21-RDM-4335-prerelease-2",
-    "@hmcts/ccpay-web-component": "~2.0.50",
```

Comment:
there's a reference to `ccpay-web-component` also in toolkit. Can you double check whether we have to update also there?  
https://github.com/hmcts/ccd-case-ui-toolkit/blob/c10b9b5bf98754e9f9c938bafc40eaef05ae42f3/package.json#L80

Reply:
Checked with pay team and @hemantt - shouldn't be required.